packages/ui/src/components/navbar-new/app-sidebar.tsx

```diff
@@ -0,0 +1,219 @@
+              />
+            </SearchProvider>
+          ) : (
+            <SidebarSearch
+              logo={
+                <Link className="flex items-center gap-1.5" to="/">
+                  <Icon name="harness" size={18} className="text-foreground-accent" />
+                  <Icon name="harness-logo-text" width={65} height={15} className="mb-0.5 text-foreground-1" />
+                </Link>
+              }
+            />
+          )}
+        </Sidebar.Header>
+        <Sidebar.Content>
+          <Sidebar.Group className="px-4 pt-5">
+            <Sidebar.GroupContent>
+              <Sidebar.Menu>
+                {pinnedMenuItems.map((item, index) => (
+                  <SidebarItem
+                    item={item}
+                    key={index}
+                    handleChangePinnedMenuItem={handleChangePinnedMenuItem}
+                    handleRemoveRecentMenuItem={handleRemoveRecentMenuItem}
+                    handleCustomNav={handleCustomNav}
+                    t={t}
+                  />
+                ))}
+
+                <Sidebar.MenuItem className="cursor-pointer">
+                  <Sidebar.MenuButton asChild onClick={handleMoreMenu}>
+                    <div>
+                      <Icon
+                        name="ellipsis"
+                        size={12}
+                        className="text-icons-4 transition-colors hover:text-primary !w-[12px] !h-[auto]"
+                      />
+                      <span className="text-foreground-3 font-medium transition-colors hover:text-primary">More</span>
+                    </div>
+                  </Sidebar.MenuButton>
+                </Sidebar.MenuItem>
+              </Sidebar.Menu>
+            </Sidebar.GroupContent>
+          </Sidebar.Group>
+
+          {!!recentMenuItems.length && (
+            <Sidebar.Group title="Recent" className="border-t px-4 pt-3">
+              <Sidebar.GroupLabel>Recent</Sidebar.GroupLabel>
+              <Spacer size={2} />
+              <Sidebar.GroupContent>
+                <Sidebar.Menu>
+                  {recentMenuItems.map(item => (
+                    <SidebarItem
+                      isRecent
+                      key={item.id}
+                      item={item}
+                      handleChangePinnedMenuItem={handleChangePinnedMenuItem}
+                      handleRemoveRecentMenuItem={handleRemoveRecentMenuItem}
+                      handleCustomNav={handleCustomNav}
+                      t={t}
+                    />
+                  ))}
+                </Sidebar.Menu>
+              </Sidebar.GroupContent>
+            </Sidebar.Group>
+          )}
+
+          <Sidebar.Group className="border-t px-4 pt-5">
+            <Sidebar.GroupContent>
+              <Sidebar.Menu>
+                {!!currentUser?.admin && (
+                  <Sidebar.MenuItem>
+                    <Sidebar.MenuButton
+                      asChild
+                      className="cursor-pointer"
```

Comment:
```suggestion

```
I think we can set this class as the default for this component, as it is interactive in nature, so we don't have to pass it every time.